README.md

```diff
@@ -416,17 +317,17 @@ status:
     message: "resource is failed"
 ```
 
-- `observedGeneration == metadata.generation`, so kubernetes-deploy will check this resource's success and failure conditions.
+- `observedGeneration == metadata.generation`, so krane will check this resource's success and failure conditions.
 - Since `$.status.conditions[?(@.type == "Ready")].status == "False"`, the resource is not considered successful yet.
 - `$.status.conditions[?(@.type == "Failed")].status == "True"` means that a failure condition has been fulfilled and the resource is considered failed.
-- Since `error_msg_path` is specified, kubernetes-deploy will log the contents of `$.status.conditions[?(@.type == "Failed")].message`, which in this case is: `resource is failed`.
+- Since `error_msg_path` is specified, krane will log the contents of `$.status.conditions[?(@.type == "Failed")].message`, which in this case is: `resource is failed`.
 
 ### Deploy walkthrough
 
 Let's walk through what happens when you run the `deploy` task with [this directory of templates](https://github.com/Shopify/kubernetes-deploy/tree/master/test/fixtures/hello-cloud). You can see this for yourself by running the following command:
 
 ```bash
-krane deploy my-namespace my-k8s-cluster -f test/fixtures/hello-cloud --render-erb
+krane render -f test/fixtures/hello-cloud --current-sha 1| krane deploy my-namespace my-k8s-cluster  --stdin
```

Comment:
This reference to `krane render` comes out of left field and will probably be confusing to people reading through this for the first time. Minimally, I think we should provide a link to the section on `krane render` and a note/comment that, since these fixtures use ERB templates, we must first pass them through our renderer